data/json/monsters/nether.json

```
@@ -301,7 +301,7 @@
       "NOHEAD",
       "BASHES",
       "FLIES",
-      "ATTACKMON",
+      "PUSH_MON",
```

```suggestion
      "PUSH_MON",
      "PUSH_VEH",
```
This is from how the original version is depicted - wind control, sort of.